core/__tests__/models/group/rules/topLevelGroupRules.ts

```diff
@@ -19,6 +19,11 @@ describe("model/group", () => {
     toad = response.toad;
   }, helper.setupTime);
 
+  beforeAll(async () => {
+    const otherModel = await helper.factories.model({ name: "other model" });
+    await GrouparooRecord.create({ modelId: otherModel.id });
+  });
```

Comment:
This suite of test would fail if there was another record in another model.  Now fixed